protobufs/gen/swagger/api/api.swagger.json

```diff
@@ -1211,6 +1211,78 @@
                 }
             }
         },
+        "/v1/entities/{document_id}/share": {
+            "post": {
+                "description": "Shares the entity with the target identity.",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "tags": [
+                    "Entities"
+                ],
+                "summary": "Shares the entity with the target identity.",
```

Comment:
I find this summary to be a bit confusing. Maybe something like, "shares the entity data with the target identity", or "data from the entity profile", something like that. but not very opinionated, could leave also as is.

Reply:
I think, `Share gives Entity access to the target owner` would be appropriate i guess.